onnxruntime/core/providers/cpu/ml/ml_common.h

```diff
@@ -27,7 +27,19 @@ enum NODE_MODE : uint8_t {
   BRANCH_GTE = 6,
   BRANCH_GT = 8,
   BRANCH_EQ = 10,
-  BRANCH_NEQ = 12
+  BRANCH_NEQ = 12,
+  BRANCH_MEMBER = 14,
+};
+
+enum class NODE_MODE_V5 : uint8_t {
```

Comment:
Why use different values?

Reply:
In the [documentation](https://github.com/onnx/onnx/blob/main/docs/Operators-ml.md#ai.onnx.ml.TreeEnsemble), `ai.onnx.ml.TreeEnsemble.v5` has different values for `node_modes` as before they were strings. This enum is only used in the conversion from v5 to the current internal implementation.